### DISCUSSION TITLE
What is the CSS Selector for changing cursor in ckeditor?

### DISCUSSION THREAD

**Discussion by bogganj94** [TriliumNext/Trilium#2245] [Q&A]
Hello i am making a custom theme but i cant find the CSS Selector for the input cursor in CKeditor? right now its too dark and i cant see where i am when typing until i type.  the page the input cursor is dark on is the CSS file for the theme

**Comment by abitofevrything** [comment]
This isn't specific to CKEditor. You should use a selector with the `cursor` attribute set. You can then set the cursor to an image of your choice.
